packages/interactor/src/create-interactor.ts

```diff
@@ -18,8 +19,11 @@ export function createInteractor<E extends Element>(interactorName: string) {
           if(args.length) {
             actionDescription += ` with ` + args.map((a) => JSON.stringify(a)).join(', ');
           }
-          return interaction(`${actionDescription} on ${this.description}`, () => {
-            return converge(() => {
+          return interaction(`${actionDescription} on ${this.description}`, async () => {
+            if(bigtestGlobals.runnerState === 'assertion') {
```

Comment:
Is it worth making a check for `runnerState === 'pending'`? Probably not, but at the same time it definitely would be an indicator that something is not right.

Reply:
I thought about it, but this felt like it'd be enough.